### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by donizer**
- [DEMO LINK](https://donizer.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://donizer.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

**Comment by donizer** [comment]
all [here](https://donizer.github.io/layout_product-cards/) and [here]https://donizer.github.io/layout_product-cards/report/html_report/) seems to be fine, but test on github failed. don't know why.

File: src/index.html [hunk 1]
```diff
@@ -5,9 +5,50 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
```
Comment (anastasiiavorobiova) [inline src/index.html:8]: Consider adding a proper font https://fonts.google.com/specimen/Roboto 
Screenshot 2023-10-01 at 18 00 14
1. Click on the needed font weights 2. Copy the font link 3. Add the font in the head of your HTML before stylesheets

File: src/index.html [hunk 2]
```diff
@@ -5,9 +5,50 @@
...
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="monitor"
+      >
+
+      <p class="card__name">
```
Comment (anastasiiavorobiova) [inline src/index.html:18]:
```suggestion
      <h2 class="card__name">
```
Consider using a semantic tag for the title

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Looks nice! Just a few comments to fix

File: src/styles/index.scss [hunk 3]
```diff
@@ -1,3 +1,138 @@
...
+  font-family: "Roboto", sans-serif;
+}
+
+body,
+p,
+h2 {
   margin: 0;
 }
+
+.card {
```
Comment (SanyaBratashchuk) [inline src/styles/index.scss:19]: move each BEM block to the separate file

**Review by SanyaBratashchuk — APPROVED** [review]

**Pull request closed without merging**